docs/basic_training/config.pt.md

```diff
@@ -157,13 +156,13 @@ nextflow execute foo.nf -c my-env.config
 
 ### Configurar processos
 
-`process` [directives](https://www.nextflow.io/docs/latest/process.html#directives) permitem a especificação de configurações para a execução da tarefa, como `cpus`, `memory`, `container`, e outros recursos no script do pipeline.
+A [diretiva](https://www.nextflow.io/docs/latest/process.html#directives) `process`  permite a especificação de configurações para a execução de uma tarefa, como `cpus`, `memory`, `container`, além de outros recursos, no script do pipeline.
 
-Isso é útil ao criarmos um exemplo-teste ou um protótipo de fluxo de trabalho.
+Isso é útil ao criarmos um exemplo-teste ou um protótipo para o nosso pipeline.
```

Comment:
I simplified the term "workflow" to "pipeline", as I think "workflow" here means the pipeline as a whole

Reply:
🥳